staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -598,6 +601,40 @@ type DiskTarget struct {
 	PciAddress string `json:"pciAddress,omitempty"`
 }
 
+type LaunchSecurity struct {
+	// AMD Secure Encrypted Virtualization (SEV).
+	SEV *SEV `json:"sev,omitempty"`
+}
+
+type SEV struct {
+	// Guest policy flags as defined in AMD SEV API specification.
+	// Note: due to security reasons it is not allowed to enable guest debugging. Therefore NoDebug flag is not exposed to users and is always true.
+	Policy *SEVPolicy `json:"policy,omitempty"`
+}
+
+type SEVPolicy struct {
+	// Sharing keys with other guests is disallowed.
+	// Defaults to false.
+	// +optional
+	NoKeysSharing *bool `json:"noKeysSharing,omitempty"`
+	// SEV-ES is required.
+	// Defaults to false.
+	// +optional
+	EncryptedState *bool `json:"encryptedState,omitempty"`
+	// Sending the guest to another platform is disallowed.
+	// Defaults to false.
+	// +optional
+	NoSend *bool `json:"noSend,omitempty"`
+	// The guest must not be transmitted to another platform that is not in the domain.
+	// Defaults to false.
+	// +optional
+	Domain *bool `json:"domain,omitempty"`
+	// The guest must not be transmitted to another platform that is not SEV capable.
+	// Defaults to false.
+	// +optional
+	SEV *bool `json:"sev,omitempty"`
```

Comment:
All these knobs look a little bit too detailed for the initial stab for me. @vasiliy-ul do you think you can trim it down to the absolut minimum right now? 

Or maybe they make sense, but I am not sure what to do with bools like `domain` or `sev`. I would expect by default that if sev is enabled, that it i encrypted and that is basically it. Is "another" platform another node? If yes, this is probably related, but it may not be the right exposure granularity.

Reply:
Then probably I will completely remove the policy flags from the VMI spec... at least for now they are not very usefull. They can be added later gradually along with the corresponding features.

Reply:
> Then probably I will completely remove the policy flags from the VMI spec... at least for now they are not very usefull. They can be added later gradually along with the corresponding features.

That sounds great. Thanks!

Reply:
Done. Removed the policy in a separate commit so if needed the logic can be restored from the git history.